app/javascript/components/admin/site_settings/appearance/DeleteBrandingImageModal.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Button, Stack } from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import useDeleteSiteSetting
+  from '../../../../hooks/mutations/admin/site_settings/useDeleteSiteSetting';
+
+export default function DeleteBrandingImageModal({ handleClose }) {
+  const deleteSiteSetting = useDeleteSiteSetting('BrandingImage');
+
+  return (
+    <>
+      <span className="text-center"> Are you sure you want to delete the branding image?</span>
+      <Stack direction="horizontal" gap={1} className="float-end mt-3">
+        <Button variant="brand-backward" onClick={handleClose}>
+          Close
+        </Button>
+        <Button variant="danger" onClick={deleteSiteSetting.mutate}>
+          Delete
+        </Button>
+      </Stack>
+    </>
+  );
+}
+
+DeleteBrandingImageModal.propTypes = {
+  handleClose: PropTypes.func,
+};
+
+DeleteBrandingImageModal.defaultProps = {
+  handleClose: () => {},
+};
```

Comment:
Why this is not consistent with its siblings `DeleteRoomForm, DeleteUserForm, DeleteRoleForm,...`?

Reply:
Because I think there is a plan to refactor most of the forms to a onClick hook.
Forms are not required for a simple request.